### DISCUSSION TITLE
Bug: Blog Post a Few Weeks in The Future

### DISCUSSION THREAD

**Discussion by versecafe** [vercel/community#7193] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/blog/first-input-delay-vs-interaction-to-next-paint

### Steps to Reproduce

@lydiahallie Not sure how you managed it but the FID blog post has a date of September 2024 on it.

<img width="1413" alt="image" src="https://github.com/vercel/community/assets/147033096/3e2c6127-9a29-43f6-8398-d0ef72fefef9">

**Comment by bitttttten** [comment]
Now this is what we call the bleeding edge.

**Reply by versecafe** [reply]
Vercel Temporal Edge: Serve your content a month before you even create it

warning may create paradoxes and destroy the timeline

**Comment by amyegan** [comment]
Thanks for reporting this!

**Reply by amyegan** [reply, marked as answer]
It's fixed now 😄

[a comment by an automated account is not shown]
